keras/src/backend/jax/math.py

```diff
@@ -123,6 +123,15 @@ def fft2(x):
     return jnp.real(complex_output), jnp.imag(complex_output)
 
 
+def ifft2(x):
```

Comment:
Please use the built-in version https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.fft.ifft2.html

Reply:
updated to use built-in version